opentok-react-native.podspec

```diff
@@ -17,5 +17,5 @@ Pod::Spec.new do |s|
   s.source_files  = "ios/**/*.{h,m,swift}"
 
   s.dependency 'React'
-  s.dependency 'OpenTok'
+  s.dependency 'OpenTok','2.18.1'
```

Comment:
Shall we update to the latest `2.19.1`?

Reply:
not as part of this release I don't think